CHANGELOG.carto.md

```diff
@@ -1,5 +1,12 @@
 # CARTO node-mapnik changelog
 
+## 3.6.2-carto.2
+
+**Release date**: 2018-01-15
```

Comment:
I would not set the release that yet, but "stub" the next release. We will probably pull more changes in that one.

Reply:
I was hoping to use carto releases instead of my own fork but I don't know if that's possible even with this fix, so I'll leave the date as 2018-XX-XX for now.